packages/enzyme-test-suite/test/_helpers/react-compat.js

```diff
@@ -37,9 +38,16 @@ if (is('^16.3.0-0')) {
   createContext = null;
 }
 
+if (is('^16.3.0-0')) {
+  ({ forwardRef } = require('react'));
```

Comment:
let's combine createContext with forwardRef, since they're in the same versions.